<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>property_control</class>
 <widget class="QWidget" name="property_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>45</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>272</width>
    <height>45</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cut">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_paste">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
